### ISSUE TITLE
unable to connect with external IP

### ISSUE THREAD

**Issue by banzoola** [mschnitzer/ark-survival-ascended-linux-container-image#76]
Not sure what im missing. I can confirm i can connect to the server on my internal IP address (not the docker IP) through the console within ARK however and runs fine. I can confirm NAT is working as this server hosts other services.  There are a couple of errors however i read these can be safely ignored and i doubt they would prevent internet access to the server.

Failed to read /etc/machine-id, error Invalid argument.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely

**Comment by mschnitzer** [comment]
I'm in the middle of a move, I'm coming back to you mid next week.

```
Failed to read /etc/machine-id, error Invalid argument.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
```

This is expected and not causing the issue. Looking into it next week.

**Comment by banzoola** [comment]
Thank you, i had a feeling the error wasn't causing the issue. Looking forward to problem solving this as im baffled on what the issue is.

**Comment by LLdaniel** [comment]
Here are some questions which might help to identify the problem:
* Is everything fine with port forwarding to your server? 
* Do you have a public IP address where your server is reachable?

**Comment by banzoola** [comment]
hi LLdaniel, thanks for getting back to me. 

I figured out the issue, it was to do with rebuinding of the NAT address because i was connecting to my external IP address from internally, i did try and tweak the nat settings to get around this however i had no luck, also the server didnt appear in the list which made me think this wasnt the issue.

I went to a friends house and the server connected no problem and showed up in the list.

Thanks
